ambry-api/src/main/java/com/github/ambry/network/RequestInfo.java

```diff
@@ -50,6 +51,25 @@ public RequestInfo(String host, Port port, SendWithCorrelationId request, Replic
     requestCreateTime = System.currentTimeMillis();
   }
 
+  /**
+   * Construct a RequestInfo with the given parameters
+   * @param host the host to which the data is meant for
+   * @param port the port on the host to which the data is meant for
+   * @param request the data to be sent.
+   * @param replicaId the {@link ReplicaId} associated with this request.
+   * @param chargeable the {@link Chargeable} associated with this request.
+   * @param creationTime the creation time of this request in msec.
+   */
+  public RequestInfo(String host, Port port, SendWithCorrelationId request, ReplicaId replicaId, Chargeable chargeable,
```

Comment:
Sorry for the confusion. Yeah, they are actually same in production code. I had to do so that it is compatible with test code. Since we are using `MockTime` in tests to mimic time movement, etc, had to use `time.milliseconds` instead of directly calling system time function.